scripts/config.js

```diff
@@ -205,26 +209,21 @@ ${chalk.dim(
 )}
 `,
       default: function() {
-        return rootUrlDefault ? rootUrlDefault : 'http://localhost:3000';
+        return 'MyMarketplace';
```

Comment:
Don't change this.
The end result is `REACT_APP_MARKETPLACE_ROOT_URL=MyMarketplace
`, which is wrong.